package.json

```
@@ -35,7 +35,7 @@
     "node": "^12 || ^14 || ^16"
```

We'll need to remove 12 since we are removing support and won't be testing on it
```suggestion
    "node": "^14 || ^16"
```